### PR TITLE
PIC-375: Create roles and policies for Offender Search to connect to …

### PR DESCRIPTION
**Pull request by jhevans**
…Elastic Search in Delius infra

### CODE REVIEW

**Comment by jhevans** [comment]
Please don't merge yet, waiting on confirmation of role names and environments

**Comment by poornima-moj** [comment]
@John Evans, instead of creating the IAM role for dev, pre-prod and prod at the sametime, can you please test the IAM role in dev namespace and follow the same for other environments?  However, the code looks fine and happy to approve. Can you raise the PR for dev namespace and lets see how that works.

**Comment by jhevans** [comment]
@poornima-moj amended as requested, dev only in this PR

File: terraform/cross-account-IAM/offender-search-dev.tf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  statement {
+    actions   = ["sts:AssumeRole"]
+    resources = ["arn:aws:iam::754256621582:cp-pcs-newtech-es-service-role-dev"]
+  }
+}
+
+resource "aws_iam_policy" "offender-search-dev-delius-elastic-search-policy" {
+  name   = "offender-search-dev-delius-elastic-search-policy"
+  policy = data.aws_iam_policy_document.offender-search-dev-kiam-trust-chain.json
```
Comment (poornima-moj) [inline terraform/cross-account-IAM/offender-search-dev.tf:54]: policy = data.aws_iam_policy_document.offender-search-dev-delius-elastic-search-policy.json

**Pull request merged**